MSBuild.Sdk.Extras/build/platforms/languageTargets/Wpf.targets

```diff
@@ -16,17 +16,20 @@
   <PropertyGroup>
     <EnableDefaultWpfItems Condition="'$(EnableDefaultWpfItems)' == ''">True</EnableDefaultWpfItems>
     <DefaultWpfPageExcludes>$(DefaultWpfPageExcludes);$(MSBuildProjectDirectory)/App.xaml</DefaultWpfPageExcludes>
+    <DefaultWpfPageExcludes>$(DefaultWpfPageExcludes);$(MSBuildProjectDirectory)/Application.xaml</DefaultWpfPageExcludes>
```

Comment:
Can you please consolidate these two lines? Just add `;$(MSBuildProjectDirectory)/Application.xaml` to the previous one.

Reply:
Done.